front/pages/api/w/[wId]/assistant/mentions/suggestions.test.ts

```diff
@@ -1,11 +1,32 @@
 import type { RequestMethod } from "node-mocks-http";
-import { describe, expect, it } from "vitest";
+import { describe, expect, it, vi } from "vitest";
 
 import { AgentConfigurationFactory } from "@app/tests/utils/AgentConfigurationFactory";
 import { createPrivateApiMockRequest } from "@app/tests/utils/generic_private_api_tests";
 
 import handler from "./suggestions";
 
+// Mock Elasticsearch
+vi.mock("@app/lib/api/elasticsearch", async (importOriginal) => {
```

Comment:
we need this mock to have the use suggestions now, as the feature flag is removed